Automattic/wp-calypso
# Checkout: Update some links to our docs

#### Changes proposed in this Pull Request

As our docs get updated, the links should too.

More info: p4jECJ-1hM-p2

#### Testing instructions

- Open a site w/o a plan
- Go to plans section
- Mouseover `Free Domain for One Year`
- Make sure the `regular price` is a hyperlink to our docs

------------------------------------------------------------

- Add a domain to cart
- Go to checkout
- Before completing checkout check the ToS
- `how your subscription works and how to cancel` should properly link to anchors on the `manage-purchases` support doc
